.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+workflows:
+  version: 2
+  build-docker-image:
+    jobs:
+      - build:
+          filters:
+            tags:
+              only: /.*/
+jobs:
+  build:
+    docker:
+      - image: docker:stable-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Set Docker image tag based on branch or git tag
+          command: |
+            if [ ! -z "${CIRCLE_TAG}" ] ; then
+              echo 'export DOCKER_TAG=${CIRCLE_TAG}' >> $BASH_ENV
+            elif [ "${CIRCLE_BRANCH}" == "master" ] ; then
+              echo 'export DOCKER_TAG=latest' >> $BASH_ENV
+            else
+              echo 'export DOCKER_TAG=$( echo ${CIRCLE_BRANCH} | cut -f 2 -d / )' >> $BASH_ENV
+            fi
+      - run:
+          name: Build Docker image
+          command: |
+            source ${BASH_ENV}
+            docker build \
+            --build-arg SONATYPE_USER=${SONATYPE_USER} \
```

Comment:
These values are not consumed by the Dockerfile and should be removed.

Reply:
You are right. The majority of other CircleCI config.yml I saw are having this unused args for docker build.